src/doc/bootstrap

```diff
@@ -125,11 +134,48 @@ Some packages that provide additional functionality are marked as
 integration of these packages into the Sage distribution.
 
 EOF
-for PKG_BASE in $(sage-package list --has-file SPKG.rst :experimental: | sort); do
+for PKG_BASE in $(sage-package list --has-file SPKG.rst :experimental: | grep -v '^sagemath_' | sort); do
     echo "* :ref:\`spkg_$PKG_BASE\`"
 done >> "$OUTPUT_INDEX"
 cat >> "$OUTPUT_INDEX" <<EOF
 
+Runtime-Detectable Features and Conditional Doctests
+----------------------------------------------------
```

Comment:
How about just "Features and Conditional Doctests" or "Features for Conditional Doctests" or even just "Features"? 

We do not need to explain what a feature is in the title. Or do you need? If not, let's make the title short.

Reply:
I've shortened it